polyglot-scala/pom.xml

```diff
@@ -195,6 +195,15 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <scala>
+            <scalafmt />
```

Comment:
I think you need to refer to the `.scalafmt.conf` here. Maybe, because it's not in the repository root directory. The format of the files doesn't match the config.

Reply:
Yup, realized that, but conf file needed change: error was:
```
Caused by: java.util.NoSuchElementException: version [expected 3.7.3 or 3.7.3]: 3.7.15
```